app/models/vim_performance_daily.rb

```diff
@@ -25,7 +25,7 @@ def self.excluded_cols_for_expressions
   def self.find(cnt, *args)
     raise "Unsupported finder value #{cnt}" unless cnt == :all
 
-    Vmdb::Deprecation.deprecation_warning(:find, :find_entries)
+    Vmdb::Deprecation.deprecation_warning(:find, :find_entries, caller(1))
```

Comment:
> I think we need to sacrifice the elegance of the catch-all delegation

...

> I don't see how to fix this without removing the delegation

Agreed :wink:

---

Specifically, I think we'd need to do the delegation "manually" for the individual method, and have the delegating-method pass `caller_backtrace` if its own caller did not.